cluster-autoscaler/cloudprovider/openshiftmachineapi/machineapi_nodegroup_test.go

```diff
@@ -424,36 +424,33 @@ func TestNodeGroupIncreaseSize(t *testing.T) {
 		initial     int32
 		maxSize     string
 		minSize     string
-		errors      bool
+		errorMsg    string
 	}
 
 	testCases := []testCase{{
 		description: "errors because delta is negative",
 		delta:       -1,
-		errors:      true,
 		initial:     3,
 		maxSize:     "10",
 		minSize:     "1",
+		errorMsg:    "size increase must be positive",
```

Comment:
shall we go `var ErrSizeIncrease = errors.New("size increase must be positive")` and reuse that?

Reply:
I would not go so far. The errors are strings-wise compared at the end.

Reply:
Yes, let's not because in some of the other test cases we're specifying sub-string matches.

Reply:
I'm not sure I follow. We could categorise our errors as `var ErrWhatEver = errors.New("size increase must be positive")` and return that from the function. Then in the tests compare the error with `ErrWhatEver` so no need to maintain duplicated strings or compare strings directly but the error itself. Not sure why that is going far

Reply:
This is all true.